ecs-cli/modules/cli/local/localproject/project.go

```diff
@@ -219,7 +219,7 @@ var readTaskDefFromRemote = func(remote string, p *LocalProject) (*ecs.TaskDefin
 	commandConfig, err := newCommandConfig(p.context, rdwr, region)
 	if err != nil {
 		logrus.WithFields(logrus.Fields{
-			"error": err,
```

Comment:
Why do we need two continuous space here?

Reply:
@iamhopaul123 see: https://golang.org/cmd/gofmt/